### DISCUSSION TITLE
【HELP!!!】Compile tirton with  triton-shared but not effect。

### DISCUSSION THREAD

**Discussion by FatJhon** [microsoft/triton-shared#84] [General]
i try to compile  triton-shared as a submodule in tirton,but not succeed.
step as followed:
1. git clone https://github.com/openai/triton.git
2. cd triton
3. git add submodule  https://github.com/microsoft/triton-shared.git third_party/triton_shared
4. export TRITON_CODEGEN_TRITON_SHARED=1
5. pip install ninja cmake wheel; pip install -e python
6. Done
tirton is installed， but i cannot find triton-shared-opt.

**Comment by nhat-nguyen** [comment]
Hi @FatJhon, this is a known issue. The latest triton repo has recently reworked the 3rd party integration. There is a related issue here: https://github.com/microsoft/triton-shared/issues/76
We are working on a fix. In the mean time, you can still play around with triton-shared by building triton at commit `56c284cf7e39f249cdf1d8d5dba7892deb0286d6` instead of latest.

**Comment by aaronsm** [comment]
@FatJhon Take a look at issue https://github.com/microsoft/triton-shared/issues/91 for how to fix the build
